subtests/docker_cli/systemd_in_container/test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#
+# Test systemd in containers
+#
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 IMAGE-NAME"
+    exit 1
+fi
+image="$1"
+
+# Exit status. Default 0 (success), but set to 1 on any subtest failure
+rc=0
+
+# Main code. Run a command whose output is expected to be a single line,
+# and the first field is a security context (user:role:type:level); verify
+# that command generates exactly one line of output; extract the <type>
+# and <level> fields; and check against a list of one or more expected
+# values. Assume that there's no overlap between valid types & ranges.
+check_output() {
+    local name="$1"; shift
+    local command="$1"; shift
+
+    $command < /dev/ptmx > /tmp/systemd$$.log 2>&1 &
+    sleep 30
```

Comment:
>The problem is that we might want to check for multiple strings in the output and some of them might already be available and some not yet .

If so, the inner loop can be rewritten or refactored as a helper function. The important point is, please don't use `sleep 30`.